packages/components/src/InputEmail/InputEmail.test.tsx

```diff
@@ -52,7 +58,9 @@ it("Doesn't show validation when you're first typing", async () => {
   const input = getByLabelText("Foo");
 
   input.focus();
-  fireEvent.change(input, { target: { value: "not an email" } });
+  act(() => {
+    fireEvent.change(input, { target: { value: "not an email" } });
+  });
```

Comment:
are these neeed in this PR?

Reply:
Remove here: https://github.com/GetJobber/atlantis/pull/627/commits/b4cdbd022b24a2caa2d55e477b72b1ea5dbcca90